src/goto-programs/remove_exceptions.cpp

```diff
@@ -116,7 +118,9 @@ void remove_exceptionst::add_exceptional_returns(
     {
       const exprt &function_expr=
         to_code_function_call(instr_it->code).function();
-      assert(function_expr.id()==ID_symbol);
+      INVARIANT(
+        function_expr.id()==ID_symbol,
```

Comment:
Likewise, is this a DATA_INVARIANT, i.e. a condition on the general well-structured-ness of the exprt?